### DISCUSSION TITLE
Terraform not updating certain folders

### DISCUSSION THREAD

**Discussion by jpealing-fiscaltec** [renovatebot/renovate#17772] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Terraform updates work fine for some paths but not others, e.g. in the following minimal reproduction:

https://github.com/jpealing-fiscaltec/renovate-terraform

Terraform in the `Infrastructure/Foo/environments/prod` folder is updated, but the otherwise identical `Infrastructure/Foo/environments/test` folder is not touched.

Logs indicate that the `Infrastructure/Foo/environments/test` folder is not even considered for updates.

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Matched 1 file(s) for manager terraform: Infrastructure/Foo/environments/prod/main.tf
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment, marked as answer]
The preset config:base includes preset :ignoreModulesAndTests which ignores all test folders. You can see this ignore list in your debug logs too.

You can search this repo for past questions and solutions.

**Reply by jpealing-fiscaltec** [reply]
Ah thank you! Just tested and that fixes it.
